demisto_sdk/__main__.py

```diff
@@ -213,11 +242,30 @@ def main(config, version, release_notes):
     is_flag=True,
     show_default=True,
 )
+@click.option(
+    "--console_log_threshold",
+    help="Minimum logging threshold for the console logger.",
+)
+@click.option(
+    "--file_log_threshold",
+    help="Minimum logging threshold for the file logger.",
+)
+@click.option(
+    "--log_file_path",
+    help="Path to the log file. Default: ./demisto_sdk_debug.log.",
+)
 @pass_config
-def split(config, **kwargs):
+@click.pass_context
+def split(ctx, config, **kwargs):
     """Split the code, image and description files from a Demisto integration or script yaml file
     to multiple files(To a package format - https://demisto.pan.dev/docs/package-dir).
     """
+    logging_setup(
+        console_log_threshold=kwargs.get("console_log_threshold") or logging.INFO,
+        file_log_threshold=kwargs.get("file_log_threshold") or logging.DEBUG,
+        log_file_path=kwargs.get("log_file_path") or None,
+    )
+    handle_deprecated_args(ctx.args)
```

Comment:
can we avoid the duplication between this and [this](https://github.com/demisto/demisto-sdk/pull/2707/files#diff-45eb742da124956f3cc885896232c8dbe59d6115557ff6fbae49ae6fb4eb1b50R158-R160)?

Reply:
Which? I guess the link got rot.